src/components/Coverage/Coverage.js

```diff
@@ -78,11 +79,17 @@ export default class Coverage extends React.Component {
 
     if (isMonograph) {
       return (
-        <MonographCoverage pci={dataToRender} />
+        <div>
+          <MonographCoverage pci={dataToRender} />
+          <Embargo embargo={dataToRender.embargo} />
+        </div>
       );
     } else {
       return (
-        <SerialCoverage statements={dataToRender.coverage} />
+        <div>
+          <SerialCoverage statements={dataToRender.coverage} />
+          <Embargo embargo={dataToRender?.embargo ?? dataToRender?._object?.embargo ?? dataToRender?.resource?._object.embargo} />
```

Comment:
This line is kinda confusing. I think you could move this logic to the individual if blocks above like how we populate the `dataToRender` value to make it a bit more cleaner and consistent? Like define a `let embargo` variable and populate them based on wether its a pci, line or an eresource.

Also the `??` needs to be used only to assign a default value such as `undefined ?? {}` or `null || {}` but not as a substitue to the `||` operator